packages/react-composites/tests/browser/call/app/index.tsx

```diff
@@ -161,4 +177,43 @@ function onFetchParticipantMenuItems(): IContextualMenuItem[] {
   ];
 }
 
+const onFetchCustomButtonProps: CustomCallControlButtonCallback[] = [
+  (args: CustomCallControlButtonCallbackArgs): CustomCallControlButtonProps => {
+    return {
+      showLabel: args.displayType !== 'compact',
+      // Some non-default icon that is already registered by the composites.
+      onRenderOffIcon: () => <Icon iconName="MessageSeen" />,
+      onRenderOnIcon: () => <Icon iconName="MessageSeen" />,
+      strings: {
+        label: 'custom #1'
+      },
+      placement: 'first'
```

Comment:
in mobile appears to not be there:
![image](https://user-images.githubusercontent.com/2684369/149996834-52170330-33c2-4b95-89f4-0d14095adb25.png)

Reply:
That snapshot is incorrect. I'm fixing the build so we get the right icon.